### ISSUE TITLE
Starting app with `SLINT_FULLSCREEN=1` fails against Weston 10 on i.mx8 Apalis board

### ISSUE THREAD

**Issue by tronical** [slint-ui/slint#4700]
```
xdg_wm_base@10: error 4: xdg_surface geometry (1280 x 800) is larger than the configured fullscreen state (0 x 0)
Protocol error 4 on object xdg_wm_base@10:
Protocol error 4 on object xdg_wm_base@10:
Protocol error 4 on object xdg_wm_base@10:
Protocol error 4 on object xdg_wm_base@10:
Protocol error 4 on object xdg_wm_base@10:
Protocol error 4 on object xdg_wm_base@10:
Protocol error 4 on object xdg_wm_base@10:
Protocol error 4 on object xdg_wm_base@10:
Protocol error 4 on object xdg_wm_base@10:
thread 'main' panicked at examples/energy-monitor/src/main.rs:49:18:
called `Result::unwrap()` on an `Err` value: Other("Error running winit event loop: Exit Failure: 1")
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

It works without fullscreen. This is a regression, as previous releases worked. Will try to bisect.

**Comment by tronical** [comment]
This happened before as #534

**Comment by tronical** [comment]
Fixed by #4723 🥳
